src/collector.ts

```diff
@@ -105,9 +105,33 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
         let goModDeps = contents.split("\n").reduce((dependencies, line, index) => {
-            // Ignore "replace" lines
-            if (!line.includes("=>")) {
+            if (line.includes("=>")) {
+                // skip any text after '//'
+                if (line.includes("//")) {
+                    line = line.split("//")[0];
+                }
+                // split the replace statements by '=>'
+                const parts: Array<string> = line.replace('replace', '').replace('(', '').replace(')', '').trim().split(' => ');
+                const replaceTo: Array<string> = (parts[0] || '').trim().split(' ');
+                const replaceWith: Array<string> = (parts[1] || '').trim().split(' ');
+                regExp.lastIndex = 0;
+                const replaceWithVersion = regExp.exec(parts[1]);
+                const replaceWithIndex = line.indexOf(parts[1]);
+
+                // Skip lines without final version string
+                if (replaceWithVersion && replaceWithVersion.length > 0) {
+                    const replaceEntry: IKeyValueEntry = new KeyValueEntry(replaceWith[0].trim(), { line: 0, column: 0 });
+                    replaceEntry.value = new Variant(ValueType.String, 'v' + replaceWithVersion[0]);
+                    replaceEntry.value_position = { line: index + 1, column: (replaceWithIndex + replaceWithVersion.index) };
+                    const replaceData = {
+                        version: (replaceTo[1] || '').trim(),
+                        replaceDependency: new Dependency(replaceEntry)
+                    }
+                    replaceDependencies.set(replaceTo[0].trim(), replaceData);
```

Comment:
This map might not work in case when we have same module with different version to be replaced by different modules.
Need to include module@version kind of key for replace map.